app/build.gradle.kts

```diff
@@ -21,6 +21,7 @@ dependencies {
     testImplementation(kotlin("test-junit"))
     testImplementation("org.junit.jupiter:junit-jupiter-api:5.1.0")
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.1.0")
+    implementation("org.jetbrains.kotlinx:kotlinx-collections-immutable-jvm:0.3.3")
```

Comment:
What's this used for?

Reply:
This was a missing library at runtime when using the connect4 .jar. But I removed kotlinx entirely from that code, and recompiled the .jar.